showcase-docs/corporate-proxy.md

```diff
@@ -0,0 +1,102 @@
+# Running the Showcase application behind a corporate proxy
+
+Out of the box, the Showcase application can be run behind a corporate proxy, by setting any of the following environment variables prior to starting the application:
+
+- `HTTP_PROXY`: HTTP proxy to use.
+- `HTTPS_PROXY`: distinct proxy to use for HTTPS requests.
+
+Additionally, you can set the `NO_PROXY` environment variable to exclude certain domains from proxying. The value is a comma-separated list of hostnames that do not require a proxy to get reached, even if one is specified.
+
```

Comment:
Would plugin vendors need to do anything here?  I assume as long as they are using one of `axios`, `fetch`, `node-fetch`, they should get this support for free?  Maybe it's worth calling out in our product docs that these are the only HTTP request clients we support.  In general, we should be following the upstream ADR that recommends the use of `node-fetch` but we don't enforce this.

Reply:
There is indeed nothing to do for plugin vendors using Axios, `fetch` or `node-fetch`. I'll mention this in the docs, and make sure it is also called out in our product docs. Thanks.

Reply:
Added in [`66a5d93` (#1225)](https://github.com/janus-idp/backstage-showcase/pull/1225/commits/66a5d930aaecf5b0ea2440a20531e19aeb63cfa2)